samples/app/FilamentApp.h

```diff
@@ -77,6 +79,13 @@ class FilamentApp {
     FilamentApp& operator=(const FilamentApp& rhs) = delete;
     FilamentApp& operator=(FilamentApp&& rhs) = delete;
 
+    // Returns the path to the Filament root for loading assets. For now this is determined by
+    // looking up from the executable folder, which allows users to launch samples from any folder.
+    static const utils::Path& getRootPath() {
+        static const utils::Path root = utils::Path::getCurrentExecutable().getAncestor(3);
```

Comment:
This assumes we always run samples from out/xxx/samples. This will break if someone builds manually with cmake or if we distribute the samples in a zip file.

Reply:
Yup, and we were already making this assumption in some places, this merely consolidates the badness.

Maybe I should add back the asset-copying stuff to CMake, which would allow me to replace getAncestor with getParent...

Reply:
I think that's the best solution.

Reply:
Done.